common/test/acceptance/pages/lms/course_wiki.py

```diff
@@ -0,0 +1,81 @@
+"""
+Wiki tab on courses
+"""
+
+from .course_page import CoursePage
+from ...pages.studio.utils import type_in_codemirror
+
+
+class CourseWikiPage(CoursePage):
+    """
+    Course wiki navigation and objects.
+    """
+
+    url_path = "wiki"
+
+    def is_browser_on_page(self):
+        """
+        Browser is on the wiki page if the wiki breadcrumb is present
+        """
+        return self.q(css='.breadcrumb').present
+
+    def open_editor(self):
+        """
+        Replace content of a wiki article with new content
+        """
+        edit_button = self.q(css='.fa-pencil')
+        edit_button.click()
```

Comment:
I see.  Not required for this PR, but do you think it makes sense to pull out the wait for logic into a method of CourseWikiTest?